st2common/st2common/models/api/rule.py

```diff
@@ -162,7 +163,7 @@ def to_model(cls, rule):
 
         trigger = reference.get_str_resource_ref_from_model(trigger_db)
         criteria = dict(getattr(rule, 'criteria', {}))
-        pack = str(rule.pack)
+        pack = getattr(rule, 'pack', DEFAULT_PACK_NAME)
```

Comment:
Isn't action defaults to the pack which contains it? Which was handy for creating them in one pack (e.g. default) and moving to another without changing the metadata files. 
I think we should apply the same logic to all types of content.